tests/bdd/features/rebuild/test_log_based_rebuild.py

```diff
@@ -0,0 +1,321 @@
+"""Partial Rebuild feature tests."""
+
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+
+import pytest
+import http
+import os
+import subprocess
+import time
+from urllib.parse import urlparse
+from retrying import retry
+
+from common.deployer import Deployer
+from common.apiclient import ApiClient
+from common.docker import Docker
+from common.fio import Fio
+
+from openapi.model.create_pool_body import CreatePoolBody
+from openapi.model.create_volume_body import CreateVolumeBody
+from openapi.model.publish_volume_body import PublishVolumeBody
+from openapi.model.protocol import Protocol
+from openapi.exceptions import ApiException
+from openapi.model.volume_status import VolumeStatus
+from openapi.model.volume_policy import VolumePolicy
+from openapi.model.pool_status import PoolStatus
+from openapi.model.replica_state import ReplicaState
+from openapi.model.child_state import ChildState
+
+VOLUME_UUID = "5cd5378e-3f05-47f1-a830-a0f5873a1449"
+VOLUME_SIZE = 10485761
+NUM_VOLUME_REPLICAS = 3
+NODE_1_NAME = "io-engine-1"
+NODE_2_NAME = "io-engine-2"
+NODE_3_NAME = "io-engine-3"
+NODE_4_NAME = "io-engine-4"
+POOL_1_UUID = "4cc6ee64-7232-497d-a26f-38284a444980"
+POOL_2_UUID = "91a60318-bcfe-4e36-92cb-ddc7abf212ea"
+POOL_3_UUID = "4d471e62-ca17-44d1-a6d3-8820f6156c1a"
+POOL_4_UUID = "d5c5e3de-d77b-11ed-afa1-0242ac120002"
+RECONCILE_PERIOD_SECS = 1
+FIO_RUNTIME = 120
+faulted_child_uri = None
+
+
+@pytest.fixture(autouse=True)
+def init(disks):
+    Deployer.start(
+        io_engines="4",
+        wait="10s",
+        reconcile_period=f"{RECONCILE_PERIOD_SECS}s",
+        faulted_child_wait_period="30s",
```

Comment:
can we reduce this time so the test doesn't take too long?

Reply:
Changed to 15secs. For bare containers this should be enough, but haven't run tests on k8s cluster yet so will be deduced experimentally.